lib/fluent/plugin/in_syslog.rb

```diff
@@ -141,12 +145,13 @@ def multi_workers_ready?
     def start
       super
 
-      log.info "listening syslog socket on #{@bind}:#{@port} with #{@protocol_type}"
-      case @protocol_type
+      log.info "listening syslog socket on #{@bind}:#{@port} with #{@protocol_type || @transport_config.protocol}"
+      case @protocol_type || @transport_config.protocol
```

Comment:
Default `@transport_config.protocol` is `tcp`. Change default to `udp`.

Reply:
good catch 🙏 
 https://github.com/fluent/fluentd/pull/2399/commits/f23566a50b6300bab4d029e9b0ad26db36eb036a